setup.py

```diff
@@ -4,11 +4,14 @@
 from setuptools import setup, find_packages
 from os         import path
 
+import cvmfs
+
+
 readme_path = path.join(path.dirname(__file__), 'README')
 
 setup(
-  name="python-cvmfsutils",
-  version="0.2.0",
+  name=cvmfs.package_name,
+  version=cvmfs.version,
```

Comment:
well... no, that's the point.

`setup.py` includes `__init__.py` includes `whitelist.py` includes `M2Crypto` and crashes.